### DISCUSSION TITLE
Move express to dev dependency

### DISCUSSION THREAD

**Discussion by Its-Just-Nans** [node-saml/node-saml#154] [General]
Hello,

Why `express` is a `dependencies` and not a `devDependencies` ?

It's only used for test

**Comment by cjbarth** [comment]
It is used in `types.ts`.

**Reply by Its-Just-Nans** [reply]
It's looks like it is used in `test/types.ts` and for `samlTests.spec.ts`

So only for test

**Comment by markstos** [comment]
I agree this could be moved to a dev dependency because it's only used in a tests. PR created: 

https://github.com/node-saml/node-saml/pull/161
